doc/developers/maintainer.rst

```diff
@@ -81,10 +128,32 @@ Making a release
 
        $ python setup.py sdist
 
+   - You can also test a binary dist build using::
+
+       $ python setup.py bdist_wheel
+
+   - You can test if PyPi is going to accept the package using::
+
+       $ twine check dist/*
+
+   You can run ``twine check`` after step 5 (fetching artifacts) as well.
+
    The result should be in the `dist/` folder. We will upload it later
    with the wheels. Check that you can install it in a new virtualenv and
    that the tests pass.
 
+4. Proceed with caution. Ideally, tags should be created when you're almost
+   certain that the release is ready, since adding a tag to the main repo can
+   trigger certain automated processes. You can test upload the ``sdist`` to
+   ``test.pypi.org``, and test the next step by setting ``BUILD_COMMIT`` to the
+   branch name (``0.22.X`` for instance) in a PR to the wheel building repo.
+   Once all works, you can proceed with tagging. Create the tag and push it (if
+   it's an RC, it can be ``0.xxrc1`` for instance)::
```

Comment:
I understand the tagging should be done in the 0.99.X branch?

Reply:
yes, correct.